Applications/SimpleShot/layers/centering.cpp

```diff
@@ -0,0 +1,538 @@
+  1.4343022,     0.01953081,    1.9411596,     0.74064183,    0.09947904,
+  0.019532217,   0.81428003,    0.18779537,    0.7821056,     0.22682965,
+  0.10047957,    0.8144042,     0.95030487,    0.18364403,    0.77641445,
+  1.7235707,     0.839321,      0.045783173,   0.42685568,    1.2732718,
+  0.59877414,    0.9114758,     1.394809,      1.7886839,     0.16196804,
+  0.9087781,     1.439043,      0.8201566,     0.38710612,    1.7799432,
+  1.2504548,     1.931166,      0.9619099,     0.7093321,     0.8574386,
+  1.7389363,     1.7738266,     0.5942323,     0.98935664,    0.14624359,
+  0.025684139,   0.74769145,    0.8886834,     0.16716743,    0.099468775,
+  0.9984016,     0.28647056,    1.7504172,     0.7283401,     1.7715663,
+  1.7102811,     1.2373383,     0.6759054,     0.67851096,    0.11890821,
+  1.2330863,     1.7109627,     0.36601678,    0.2199844,     0.5339976,
+  0.8831787,     0.25676778,    1.6359069,     0.42123708,    1.0894456,
+  0.07940138,    1.4763635,     0.22995643,    1.2943021,     0.31505576,
+  0.6088851,     0.11047842,    0.9790464,     1.3185889,     0.060653042,
+  0.22894095,    0.04927772,    1.7385803,     0.08789811,    0.29942873,
+  0.3491881,     0.05197402,    0.18416277,    1.6313312,     0.79583687,
+  0.59852916,    0.043723065,   1.7663732,     0.9530521,     0.7741182,
+  0.7302737,     0.06661618,    0.58151186,    0.42840588,    1.8000456,
+  1.4240903,     1.695747,      1.8375303,     0.20420454,    0.8943256,
+  0.964572,      0.39459047,    0.11630259,    0.8998442,     0.72175163,
+  0.98845047,    0.24056484,    0.9729437,     0.17645803,    0.661282,
+  0.69930977,    0.9873423,     1.9364699,     0.21603872,    0.9387982,
+  1.7815027,     0.8452394,     0.37969667,    0.75867903,    0.49278417,
+  0.4124414,     0.7144215,     0.21636294,    0.7865215,     1.9734006,
+  0.33272538,    0.09349442,    0.9450515,     0.804148,      0.30519333,
+  1.5947547,     0.8924652,     0.8316595};
+
+constexpr float cifar100_conv4[64] = {
+  0.8069815,  1.0025251,  1.0039499,  1.3062875,  0.8623087,  0.8113235,
+  1.6632943,  1.8957489,  1.3761712,  0.8006514,  0.9858221,  1.4450859,
+  1.2456608,  1.5908684,  1.3463796,  1.781804,   1.2539327,  1.0488071,
+  0.69456387, 0.7837838,  1.420064,   0.96974546, 0.7254165,  0.91133595,
+  0.7480144,  1.6793756,  1.2010552,  0.37359616, 0.8411831,  0.94810975,
+  1.081522,   1.4002662,  0.91972107, 1.0329305,  1.4743291,  0.9981039,
+  1.0227196,  0.352504,   1.8154447,  1.3830396,  0.8657597,  1.4396735,
+  1.5980465,  0.99679625, 1.0142295,  1.4402367,  0.38848996, 0.9896359,
+  0.59641224, 1.1450837,  0.9463809,  0.618432,   0.6541309,  0.8063583,
+  1.6206524,  1.840674,   1.6155039,  1.1257526,  0.61734056, 0.6324227,
+  1.2998347,  2.1847007,  0.4944939,  1.3934348};
+
+namespace {
+nntrainer::Tensor getHardCodedFeatureVector(const std::string &model,
+                                            const std::string &dataset) {
```

Comment:
We have to remove this hard coded data and functions.

Reply:
I'll remove this as well

Reply:
Done